static/app/routes.tsx

```diff
@@ -72,9 +72,12 @@ type ComponentType = React.ComponentType<any>;
 export function makeLazyloadComponent<C extends ComponentType>(
   resolve: () => PromisedImport<C>
 ) {
-  return (componentProps: React.ComponentProps<C>) => {
-    return <SafeLazyLoad component={resolve} {...componentProps} />;
+  // XXX: Assign the component to a variable so it has a displayname
+  const RouteLazyLoad: React.FC<React.ComponentProps<C>> = props => {
+    return <SafeLazyLoad {...props} component={resolve} />;
```

Comment:
Was the order of this the source of the bug?

Reply:
Yeah, check the PR description. it was rendering the same component twice because the `component` prop was being passed from the parent route

![image](https://user-images.githubusercontent.com/1421724/171319201-bc0bf27f-1465-43f5-9fb6-8b1498583477.png)